packages/styled-components/src/models/ServerStyleSheet.js

```diff
@@ -26,7 +26,7 @@ export default class ServerStyleSheet {
   _emitSheetCSS = (): string => {
     const css = this.instance.toString();
     const nonce = getNonce();
-    const attrs = [nonce && `nonce="${nonce}"`, SC_ATTR, `${SC_ATTR_VERSION}="${SC_VERSION}"`];
+    const attrs = [nonce && `nonce="${nonce}"`, `${SC_ATTR}="${SC_ATTR}"`, `${SC_ATTR_VERSION}="${SC_VERSION}"`];
```

Comment:
maybe just make this `${SC_ATTR}="true"`

Reply:
sure, though not that's not strictly according to spec if I understand correctly? Works though.

https://www.w3.org/TR/xhtml1/ 4.5.1 

for example
checked="checked"